server/controllers/event.controller.js

```diff
@@ -60,9 +48,8 @@ module.exports = {
 
   destroy(req, res) {
     Event.findById(req.params.id).then(event => {
-        if (event.dataValues.owner !== req.decoded.id || !event) {
-          return res.status(404).send(messages.eventNotFound);
-        }
+        (event.dataValues.owner !== req.decoded.id || !event) &&
+          res.status(404).json(messages.eventNotFound);
         return event
```

Comment:
add console.log and check if after response console.log write message in console if yes this is defect and use

1
--
-        if (event.dataValues.owner !== req.decoded.id || !event) {
  |   | -          return res.status(404).send(messages.eventNotFound);
  |   | -        }